applications/fishing-map/src/features/map/MapScreenshot.tsx

```diff
@@ -15,21 +11,31 @@ declare global {
   }
 }
 
-const isChrome = !!window.chrome && (!!window.chrome.webstore || !!window.chrome.runtime)
-const isPrintSupported = isChrome
-
 type PrintSize = {
   px: number
   in: string
 }
 
-function MapScreenshot() {
-  const { screenshotMode, setScreenshotMode } = useScreenshotConnect()
-  const { setScreenshotLoading } = useScreenshotLoadingConnect()
-  const mapboxRef = useMapboxRef()
-  const documentFocus = useRef<boolean>(true)
+const isChrome = !!window.chrome && (!!window.chrome.webstore || !!window.chrome.runtime)
```

Comment:
I have some doubts about this:
- won't this yield false positives?
- do we correctly detect other Blink browsers such as Edge or Opera?
I would perhaps favor a more future proof solution ie using a 3rd party library (https://github.com/lancedikson/bowser for instance)

Reply:
👍